packages/@ember/template-compiler/lib/template.ts

```diff
@@ -242,12 +242,10 @@ export function template(
   const normalizedOptions = compileOptions(options);
   const component = normalizedOptions.component ?? templateOnly();
 
-  queueMicrotask(() => {
```

Comment:
TODO: for @NullVoxPopuli 
- https://github.com/emberjs/rfcs/pull/931
- does the RFC say anything about this?


Potential use case:
- forward references (like functions)